TwitchLib.Api.Helix.Models/Bits/ExtensionBitsProducts/ExtensionBitsProduct.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TwitchLib.Api.Helix.Models.Bits.ExtensionBitsProducts
+{
+    public class ExtensionBitsProduct
+    {
+        [JsonProperty(PropertyName = "sku")]
+        public string Sku { get; protected set; }
+        [JsonProperty(PropertyName = "cost")]
+        public Cost Cost { get; protected set; }
+        [JsonProperty(PropertyName = "in_development")]
+        public bool InDevelopment { get; protected set; }
+        [JsonProperty(PropertyName = "display_name")]
+        public string DisplayName { get; protected set; }
+        [JsonProperty(PropertyName = "expiration")]
+        public DateTime Expiration { get; protected set; }
```

Comment:
Does DateTime parse RFC3339 reliably?

Reply:
It does. A few models currently using it: GetStreams.Stream, CreateStreamMarker.CreatedMarker, Bits.Cheermote